spec/models/blood_sugar_spec.rb

```diff
@@ -10,4 +10,28 @@
     it { should belong_to(:patient).optional }
     it { should belong_to(:user).optional }
   end
+
+  describe '#diabetic?' do
+    [{ blood_sugar_type: :random, blood_sugar_value: 300 },
+     { blood_sugar_type: :fasting, blood_sugar_value: 200 },
+     { blood_sugar_type: :post_prandial, blood_sugar_value: 301 }].each do |row|
+      it 'returns true if blood sugar is in a high state' do
+        blood_sugar = create(:blood_sugar,
+                             blood_sugar_type: row[:blood_sugar_type],
+                             blood_sugar_value: row[:blood_sugar_value])
+        expect(blood_sugar).to be_diabetic
+      end
+    end
+
+    [{ blood_sugar_type: :random, blood_sugar_value: 299 },
+     { blood_sugar_type: :fasting, blood_sugar_value: 150 },
```

Comment:
How about 199 (and 299 below) to be consistent?

Reply:
Same as above